app/policies/application_policy.rb

```diff
@@ -52,26 +52,10 @@ def is_admin?
   end
 
   def same_org?
-    case record
-    when CasaOrg
-      user.casa_org == record
-    when CasaAdmin, CasaCase, Volunteer, Supervisor, HearingType, ContactTypeGroup, ContactTopic
-      user.casa_org == record.casa_org
-    when CourtDate, CaseContact, CaseAssignment
-      user.casa_org == record&.casa_case&.casa_org
-    when LearningHour
-      user.casa_org == record&.user&.casa_org
-    when ChecklistItem
-      user.casa_org == record&.hearing_type&.casa_org
-    when ContactType
-      user.casa_org == record&.contact_type_group&.casa_org
-    when Followup
-      user.casa_org == record&.case_contact&.casa_case&.casa_org
-    when Class # Authorizing against collection, does not belong to org
-      true
```

Comment:
This allowed thinking that a collection was authorized, when really it should have used a Policy::Scope instead of ending up here!

Reply:
I should probably add this case to app policy spec.